From 9d4ba04e35b3b0863a37fce443704f1a038e74e1 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Mon, 20 Dec 2021 17:33:29 -0500
Subject: [PATCH 2/7] sm8350-common: Optimize native executables for Cortex-A76
 CPU

 * Allow the compiler to take advantage of ARMv8.2 extensions
   to improve performance in native executables, not
   just Java code compiled by ART.

Change-Id: Idd3414de107f47a18c2769dc930e6c4bceacdc62
---
 BoardConfigCommon.mk | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 7d3de29c..47d753b4 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -25,15 +25,13 @@ TARGET_ARCH := arm64
 TARGET_ARCH_VARIANT := armv8-2a
 TARGET_CPU_ABI := arm64-v8a
 TARGET_CPU_ABI2 :=
-TARGET_CPU_VARIANT := generic
-TARGET_CPU_VARIANT_RUNTIME := kryo385
+TARGET_CPU_VARIANT := cortex-a76
 
 TARGET_2ND_ARCH := arm
 TARGET_2ND_ARCH_VARIANT := armv8-a
 TARGET_2ND_CPU_ABI := armeabi-v7a
 TARGET_2ND_CPU_ABI2 := armeabi
-TARGET_2ND_CPU_VARIANT := generic
-TARGET_2ND_CPU_VARIANT_RUNTIME := kryo385
+TARGET_2ND_CPU_VARIANT := cortex-a76
 
 # Audio
 AUDIO_FEATURE_ENABLED_EXTENDED_COMPRESS_FORMAT := true
-- 
2.49.0

